src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -68,22 +39,16 @@ const DeleteWorkspaceModal = ({ workspace, workspace: { workspace: { namespace,
       }, ['Google Cloud Bucket']) :
         'Azure Storage Container',
       ' and all its data',
-      workspaceBucketUsageInBytes !== undefined && span({ style: { fontWeight: 600 } }, ` (${Utils.formatBytes(workspaceBucketUsageInBytes)})`),
+      workspaceBucketUsageInBytes !== null && span({ style: { fontWeight: 600 } }, ` (${Utils.formatBytes(workspaceBucketUsageInBytes)})`),
```

Comment:
This had to switch to a check against null given the change here https://github.com/DataBiosphere/terra-ui/blob/842e9234dbd62699c8ae4308d5e350de88e7ab28/src/pages/workspaces/workspace/delete-workspace-modal.state.ts#L103